core/src/main/java/org/frankframework/jdbc/JdbcTransactionalStorage.java

```diff
@@ -339,6 +334,10 @@ public void configure() throws ConfigurationException {
 		if (StringUtils.isNotEmpty(getHostField())) {
 			host=Misc.getHostname();
 		}
+		if (isOnlyStoreWhenMessageIdUnique() && StringUtils.isBlank(getSlotId())) {
+			throw new ConfigurationException("'slotId' has to be configured when [onlyStoreWhenMessageIdUnique]=[true]");
+
+		}
```

Comment:
A blank slot-id in combination with only storing messages with unique ID used to result in invalid queries.
So I made it a requirement to have this always set in combination.

I could drop the requirement and fix the queries we now have, adding extra tests.

Reply:
Hier eingelijk ook met `[..]`